plugins/modules/zos_data_set.py

```diff
@@ -247,6 +247,18 @@
         that is not available, then the value C(TMPHLQ) is used.
     required: false
     type: str
+  force:
+    description:
+      - Specifies that the data set can be shared with others during a member
+        delete operation which results in the data set you are updating to be
+        simultaneously updated by others.
+      - This is helpful when a data set is being used in a long running process
```

Comment:
I think this doc came from zos_blockinfile which is slightly different in its use with `force` where that force is applied to the data set in its entirety for read and write. Where here ware only supporting member deletes as DISP=SHR.

So maybe alter this:
```
      - This is helpful when a data set is being used in a long running process
        such as a started task and you are wanting to update or read.
```
to 
```
      - This is helpful when a data set is being used in a long running process
        such as a started task and you delete a member.

Reply:
i kept the "are wanting to..." portion from the copied section unless you specifically omitted it